src/scripts/main.js

```diff
@@ -21,7 +21,23 @@
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
```

Comment:
Using `for...in` is correct here, but be aware that it iterates over all enumerable properties, including those inherited through the prototype chain. Since you are using `hasOwnProperty`, this is safe, but consider using `Object.keys(data)` for a more modern approach.